sdk/documenttranslation/azure-ai-documenttranslation/azure/ai/documenttranslation/_models.py

```diff
@@ -157,30 +163,36 @@ def _to_generated_list(batch_document_inputs):
 
 
 class JobStatusResult(object):  # pylint: disable=useless-object-inheritance, too-many-instance-attributes
-    """Job status response.
+    """Status information about the translation job.
 
-    :ivar id: Required. Id of the job.
-    :vartype id: str
-    :ivar created_on: Required. Operation created date time.
+    :ivar str id: Id of the job.
+    :ivar created_on: The date time when the translation job was created.
     :vartype created_on: ~datetime.datetime
-    :ivar last_updated_on: Required. Date time in which the operation's status has been
-     updated.
+    :ivar last_updated_on: The date time when the translation job's status was last updated.
     :vartype last_updated_on: ~datetime.datetime
-    :ivar status: Required. List of possible statuses for job or document. Possible values
-     include: "NotStarted", "Running", "Succeeded", "Failed", "Cancelled", "Cancelling",
-     "ValidationFailed".
-    :vartype status: str
-    :ivar error: This contains an outer error with error code, message, details, target and an
-     inner error with more descriptive details.
+    :ivar str status: Status for a job.
+
+        * `NotStarted` - the job has not begun yet.
```

Comment:
A little out of this scope, but have you guys thought about making this an enum?

Reply:
This is only returned, never used as an input. If we made it an enum it would be purely for documentation purposes... last we had this conversation we decided that it should just be string.